README.md

```diff
@@ -19,6 +19,7 @@ https://github.com/user-attachments/assets/3379f79c-af90-4763-8ba5-8a8f34fd66cf
 ## Building a dongle
 
 To build a dongle yourself you can use the build guide by **carrefinho** ([prospector project](https://github.com/carrefinho/prospector)).  
```

Comment:
Maybe we can add here " based on the Seeed Studio XIAO nRF52840". This would make it more clear that both options are now supported.